src/inc/msquic.h

```diff
@@ -679,6 +680,7 @@ typedef struct QUIC_SETTINGS {
     uint8_t MaxOperationsPerDrain;
     uint8_t MtuDiscoveryMissingProbeCount;
     uint32_t DestCidUpdateIdleTimeoutMs;
+    uint8_t HyStartEnabled                  : 1;
```

Comment:
If making a new bit flags field involves a lot of work, I'd use a bigger field like uint32_t or uint64_t.

Reply:
I am sorry EcnEnabled used the last remaining bit :)

Reply:
I think we can get away with using a `uint8_t` for now and make a larger bitfield before the next release to try to stave off this problem.

Reply:
Opened #3276 to track this further